src/Faction/ui/FactionsRoot.tsx

```diff
@@ -43,10 +55,28 @@ const WorkTypesOffered = (props: { faction: Faction }): React.ReactElement => {
   );
 };
 
+const JoinChecklist = (props: { faction: Faction }): React.ReactElement => {
+  const info = props.faction.getInfo();
+  return (
+    <ul style={{ listStyleType: "none" }}>
+      {info.inviteReqs.map((condition, i) => (
+        <Typography component="li" key={i} display="flex" sx={{ alignItems: "center" }}>
+          <span style={{ display: "inline-block", marginLeft: "-2em", marginRight: "0.5em", height: "1.3em" }}>
+            {condition.isSatisfied(Player) ? (
+              <TaskAlt fontSize={fontSize} />
+            ) : (
+              <RadioButtonUnchecked fontSize={fontSize} />
+            )}
+          </span>
+          {condition.toString()}
+        </Typography>
+      ))}
+    </ul>
+  );
+};
+
```

Comment:
I just noticed this is similar to an existing `<Requirement>` element, maybe I can combine them...